lua/autorun/cl_streamcore.lua

```diff
@@ -49,6 +61,8 @@ net.Receive( "XTS_SC_StreamStop", function( len )
 end )
 
 net.Receive( "XTS_SC_StreamStart", function( len )
+    local isDisabled = streamcoreDisable:GetBool()
+    if isDisabled == true then return end
```

Comment:
a space after  this would be nice, this huge chunk of code without spaces isnt great
you could also put a space after all the local definitions